nixos/modules/programs/shadow.nix

```diff
@@ -5,6 +5,7 @@ let
 in
 {
   options = with lib.types; {
+    security.shadow.enable = lib.mkEnableOption "the shadow authentication suite" // { default = true; };
```

Comment:
This needs an explanation that it is experimental and things might not work correctly if enabled.

Also could this lock yourself out of your system?